packages/components/src/components/SegmentedControl/SegmentedControl.test.tsx

```diff
@@ -27,26 +20,31 @@ describe('SegmentedControl', () => {
   })
 
   const labels = ['segment 0', 'segment 1']
+  const a11yLabels = ['segment 0 a11y label', 'segment 1 a11y label']
```

Comment:
Wonder if this should only override one or the other to validate the fallback case of using `labels` when there's a falsy value in the array.